src/docs/release/breaking-changes/annotations-return-local-position-relative-to-object.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Annotations return local position relative to object
+description: Provide mouse events with reliable and meaningful local positions.
+---
+
+## Summary
+
+The local position returned in an annotation search is now relative to the
+region that contains the annotation object instead of the layer. This majorly
+affects mouse events (`onEnter`, `onExit`, and `onHover`).
+
+## Context
+
+Annotations are metadata that are assigned during the rendering phase to
+regions on the screen. Searching the annotations with a location gives the
+contextual information that contains that location. They are used to detect
+mouse events and the theme of app bars.
+
+When `localPosition` was first added to the search result, it was defined as
+relative to the layer that owned the annotation, which turned out to be a
+design mistake. The offset from the layer is meaningless and unreliable. For
+example, a `Transform` widget will draw on the same layer with an offset if
+the transform matrix is a simple translation, or push a dedicated
+`TransformLayer` if the matrix is non-trivial. The former case will keep the
+previous coordinate origin (for example, the top left corner of the app),
+while the latter case will move the position origin since it's on a new
+layer, and the difference of them might just be an unnoticeable scale of
+99%.
+
+Moreover, the major (if not only) users of `AnnotatedRegionLayer` are render
```

Comment:
This is written in a very defensive tone. The only people who will be reading this are people who have been affected. So we should start from the assumption that this had an effect, and work from there. Assume someone was broken by this. Why would they be broken? What can they do about it? etc.